cmd/uhc/whoami/cmd.go

```diff
@@ -17,14 +17,17 @@ limitations under the License.
 package whoami
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 
+	"github.com/openshift-online/uhc-cli/pkg/dump"
```

Comment:
This should be grouped with the other project packages

Reply:
Done